examples/complete/main.tf

```diff
@@ -270,6 +271,9 @@ module "dedicated_host" {
 #############################################################################
 
 module "slz_vsi_dh" {
+  # depends_on                    = [module.dedicated_host]
+  # dedicated_host_id               = module.dedicated_host.dedicated_host_ids[0]
```

Comment:
you should uncomment these lines if you want the example to support setting `enable_dedicated_host` to true. Just make sure if it is false, that `null` is passed for `dedicated_host_id`

Reply:
Done